data.json

```diff
@@ -1,4 +1,11 @@
 [
+  
+  {
```

Comment:
Please add yourself to the bottom :smile:

Reply:
Done!
Updated the required setup.